Content.Shared/Item/ItemToggle/ItemToggleSystem.cs

```diff
@@ -204,55 +217,56 @@ private void UpdateVisuals(Entity<ItemToggleComponent> ent)
     /// <summary>
     /// Used for items that require to be wielded in both hands to activate. For instance the dual energy sword will turn off if not wielded.
     /// </summary>
-    private void TurnOffonUnwielded(EntityUid uid, ItemToggleComponent itemToggle, ItemUnwieldedEvent args)
+    private void TurnOffOnUnwielded(Entity<ItemToggleComponent> ent, ref ItemUnwieldedEvent args)
     {
-        if (itemToggle.Activated)
-            TryDeactivate(uid, args.User, itemToggle: itemToggle);
+        TryDeactivate((ent, ent.Comp), args.User);
     }
 
     /// <summary>
     /// Wieldable items will automatically turn on when wielded.
     /// </summary>
-    private void TurnOnonWielded(EntityUid uid, ItemToggleComponent itemToggle, ref ItemWieldedEvent args)
+    private void TurnOnOnWielded(Entity<ItemToggleComponent> ent, ref ItemWieldedEvent args)
     {
-        if (!itemToggle.Activated)
-            TryActivate(uid, itemToggle: itemToggle);
+        // FIXME: for some reason both client and server play sound
```

Comment:
dew it, you prob messed up updating comp.Predictable (which seems like an unnecesary field to have tbh)

Reply:
welders need to disable prediction and i couldnt fix the wield thing mispredicting dont know why